psh/date/date.c

```diff
@@ -0,0 +1,223 @@
+/*
+ * Phoenix-RTOS
+ *
+ * date - print or set the system date and time
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Mateusz Niewiadomski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <sys/time.h>
+#include <unistd.h>
+
+#include "../psh.h"
+
+
+enum mode { mode_skip,
+	mode_help,
+	mode_print,
+	mode_set,
+	mode_parse };
+
+
+static void psh_date_info(void)
+{
+	printf("print/set the system date and time");
```

Comment:
No. That is standard for `psh_<app>_info()` functions. 

`help` command takes care of formatting/line breaking so it requires not formatted string to be passed.

Reply:
uch, if the outside function does the actual formatting, probably the interface should be to return const char* from here instead of direct printing.

Marking as resolved as it's out of scope of this PR.